### DISCUSSION TITLE
Execute command with spaces

### DISCUSSION THREAD

**Discussion by SuperCuber** [casey/just#741] [Q&A]
How do I execute a command with spaces in the name (on windows)? This is my justfile:

```make
set shell := ["cmd.exe", "/c"]

run:
	cargo build
	"D:\Program Files\WinBoard-4.8.0\WinBoard\winboard.exe" ....
```

But this doesn't work, saying `"D:\Program Files\WinBoard-4.8.0\WinBoard\winboard.exe"` is not a command - seems ike it also takes the quotes.

I tried
```make
set shell := ["cmd.exe", "/c"]

winboard := "D:\\Program Files\\WinBoard-4.8.0\\WinBoard\\winboard.exe"

run:
	cargo build
	{{winboard}} ...
```
But this doesn't work either, saying `D:\Program` is not a command.

**Comment by casey** [comment]
I'm surprised that your first example didn't work. Although `cmd.exe` quoting is quite strange. I've had a lot of issues getting seemingly simple things working.

If you open up a `cmd.exe` terminal, can you successfully run the command? And when you do so, how do you type it?

**Reply by SuperCuber** [reply]
Yep it runs exactly the same. An easier example is:
```make
run:
    cmd.exe
````
vs
```make
run:
    "cmd.exe"
```
Of course pasting both in cmd works,  but the second one doesn't work with Just.

To me it seems like it's on Just's side - for some reason, the quotes are included in the argument given to the shell as literal quotes.

**Comment by casey** [comment]
Interesting! This is definitely a bug, I created issue #743 to track it.
